### ISSUE TITLE
Rename After Bootcamp23

### ISSUE THREAD

**Issue by JeffCarver** [INTERSECT-training/packaging#1]
Rename this repository to "Packaging and Distribution"

**Comment by henryiii** [comment]
IMO that's somewhat implied and makes the URL a lot longer.

**Comment by JeffCarver** [comment]
Thanks, but @cosden and I talked and are going to try to get repository names consistent with lesson titles. We'll make a final decision after the bootcamp. Just leaving the issue here so we don't forget.

**Comment by cosden** [comment]
@henryiii, you're right. As Jeff said, let's just leave this open so we don't loose track of it. The answer might be, rename the session to just Packaging. (We're worried about things getting confusing with so many lessons and repos). But it's a post-bootcamp conversation.

**Comment by hollandjg** [comment]
- [ ] Decide whether to go with "Packaging and Distribution" or just "Packaging"
- [ ] Unify the names

**Comment by hollandjg** [comment]
we're going to remove this from the 2024 milestone – not something we want to change right now.
